great-idea-website/css/index.css

```diff
@@ -63,4 +63,101 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+/* ----Main Layout Section---- */
+#main_container{
+	margin: 0 auto;
+	display: block;
+	width: 1200px;
+  }
+  #nav{
+	display: inline-block;
+	width: 1200px;
+	height: 80px;
+  }
+  #heading{
+	display: block-inline;
+	width: 1200px;
+	height: 500px;
+	text-align: center;
+	padding-bottom: 110px;
+  }
+  #content_container{
+	display: inline-block;
+	width: 1200px;
+	height: 640px;
+  }
+  #contact_container{
+	display: block;
+	width: 1200px;
+	height: 300px;
+  }
+  #footer_container{
```

Comment:
We're using a lot of IDs directly for styling and structure - In our 1:1 we'll touch on some concepts in terms of using IDs for containing elements that can share a lot of the same properties, and then using classes to specifically style individual situations. This will help us keep our code DRY.